cmd/hub-router/startcmd/start_test.go

```diff
@@ -204,5 +233,7 @@ func tmpDir(t *testing.T) string {
 		require.NoError(t, delErr)
 	})
 
+	fmt.Println(path)
```

Comment:
left over?

Reply:
yeah, will remove it. Thanks.

Reply:
[Removed](https://github.com/trustbloc/hub-router/compare/6e6720ef520246c894439373b196331eed4506db..37ec34ef76f007d6d9cc5ff4c3cab961e323cd25).